contracts/strategies/direct-grants-simple/DirectGrantsSimpleStrategy.sol

```diff
@@ -0,0 +1,386 @@
+
+    event MILESTONE_SUBMITTED(address recipientId, uint256 milestoneId, Metadata metadata);
+    event MILESTONE_REJECTED(address recipientId, uint256 milestoneId);
+    event MILESTONES_SET(address recipientId);
+
+    /// ================================
+    /// ========== Storage =============
+    /// ================================
+
+    bool public registryGating;
+    bool public grantAmountRequired;
+    address[] private _acceptedRecipientIds;
+
+    /// @notice recipientId -> Recipient
+    mapping(address => Recipient) private _recipients;
+    /// @notice recipientId -> Milestone[]
+    mapping(address => Milestone[]) public milestones;
+    /// @notice recipientId -> Next Milestone for the recipient
+    mapping(address => uint256) public upcomingMilestone;
+    /// @notice recipientId -> Total Milestones for the recipient
+    mapping(address => uint256) public totalMilestones;
+
+    /// ===============================
+    /// ======== Constructor ==========
+    /// ===============================
+
+    constructor(address _allo, string memory _name) BaseStrategy(_allo, _name) {}
+
+    /// ===============================
+    /// ========= Initialize ==========
+    /// ===============================
+
+    function initialize(uint256 _poolId, bytes memory _data) public virtual override {
+        (bool _registryGating, bool _grantAmountRequired) = abi.decode(_data, (bool, bool));
+        __DirectGrantsSimpleStrategy_init(_poolId, _registryGating, _grantAmountRequired);
+    }
+
+    function __DirectGrantsSimpleStrategy_init(uint256 _poolId, bool _registryGating, bool _grantAmountRequired)
+        internal
+    {
+        __BaseStrategy_init(_poolId);
+        registryGating = _registryGating;
+        grantAmountRequired = _grantAmountRequired;
+        _setPoolActive(true);
+    }
+
+    /// ===============================
+    /// ============ Views ============
+    /// ===============================
+
+    /// @notice Get the recipient
+    function getRecipient(address _recipientId) external view returns (Recipient memory) {
+        return _getRecipient(_recipientId);
+    }
+
+    /// @notice Get Internal recipient status
+    /// @param _recipientId Id of the recipient
+    function getInternalRecipientStatus(address _recipientId) external view returns (InternalRecipientStatus) {
+        return _getRecipient(_recipientId).recipientStatus;
+    }
+
+    /// @notice Get recipient status
+    /// @param _recipientId Id of the recipient
+    function getRecipientStatus(address _recipientId) external view override returns (RecipientStatus) {
+        Recipient memory recipient = _getRecipient(_recipientId);
+        if (recipient.recipientStatus == InternalRecipientStatus.Accepted) {
+            return RecipientStatus.Accepted;
+        } else if (recipient.recipientStatus == InternalRecipientStatus.Rejected) {
+            return RecipientStatus.Rejected;
+        } else {
```

Comment:
```suggestion
        } else if (recipient.recipientStatus == InternalRecipientStatus.None) {
            return RecipientStatus.None;
        } else {
```

Reply:
we could improve the function by changing the order of the `InternalRecipientStatus`
```
    enum InternalRecipientStatus {
        None,
        Pending,
        Accepted,
        Rejected,
        InReview // move to bottom, so the first statuses match the RecipientStatus
    }

    function getRecipientStatus(address _recipientId) external view override returns (RecipientStatus) {
        InternalRecipientStatus internalStatus = _getRecipient(_recipientId);
        if (internalStatus == InternalRecipientStatus.InReview) {
            return RecipientStatus.Pending;
        } else {
            return RecipientStatus(uint8(internalStatus));
        }
    }
```